synapse/config/emailconfig.py

```diff
@@ -261,7 +261,7 @@ def default_config(self, config_dir_path, server_name, **kwargs):
         #   # Template directory. All template files should be stored within this
         #   # directory
         #   #
-        #   #template_dir: res/templates
+        #   #template_dir: synapse/res/templates
```

Comment:
I'm not quite sure what this is supposed to be. An example? the default?

If the latter, it's wrong... the default is to fish out the templates from within the virtualenv.